crates/lakekeeper/src/catalog/mod.rs

```diff
@@ -197,8 +189,8 @@ where
     // we're filling a auth-filtered page. Without a vec, that won't fly.
 {
     let page_size = page_size
-        .unwrap_or(DEFAULT_PAGE_SIZE)
-        .clamp(1, MAX_PAGE_SIZE);
+        .unwrap_or(CONFIG.pagination_size_default)
+        .clamp(1, CONFIG.pagination_size_max);
     let page_as_usize: usize = page_size.try_into().expect("1, 1000 is a valid usize");
```

Comment:
This is a problem now with custom specified i64s. Should it be i64s in the config or better u32 or so?

Reply:
Ok, we can keep it, but the expect is nevertheless wrong.